lib/bridge-provider.js

```diff
@@ -1,22 +1,13 @@
 const precompiledArrowhead600 = require("@rsksmart/rsk-precompiled-abis");
 
-let bridgeCache;
```

Comment:
After changing the federation and stopping previous federators, we don't want to cache the bridge contract instance to avoid issues.